scripts/generate-bootstraps.sh

```diff
@@ -382,23 +421,38 @@ while (($# > 0)); do
 done
 
 if [[ "$TERMUX_PACKAGE_MANAGER" == *" "* ]] || [[ " ${TERMUX_PACKAGE_MANAGERS[*]} " != *" $TERMUX_PACKAGE_MANAGER "* ]]; then
-	echo "[!] Invalid package manager '$TERMUX_PACKAGE_MANAGER'" 1>&2
-	echo "Supported package managers: '${TERMUX_PACKAGE_MANAGERS[*]}'" 1>&2
+	echo "[!] Invalid package manager '$TERMUX_PACKAGE_MANAGER'" >&2
```

Comment:
No specific need to remove `1` here and other places. Current way makes it explicitly clear what is being redirected to what. Will also require changes in `build-bootstrap.sh` if you do.